deepchem/models/torch_models/layers.py

```diff
@@ -6829,3 +6829,74 @@ def forward(self, x: torch.Tensor,
         coords = coords + 0.01 * coords_update
 
         return x, coords
+
+
+def cosine_dist(x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
+    """
+    Compute the cosine similarity (inner product) between two tensors.
+
+    Parameters
+    ----------
+    x: torch.Tensor
+        Input tensor of shape `(B, N, P)` representing the first set of vectors.
+    y: torch.Tensor
+        Input tensor of shape `(B, M, P)` representing the second set of vectors.
+
+    Returns
+    -------
+    torch.Tensor
+        Cosine similarity tensor of shape `(B, N, M)` where each entry represents the cosine similarity
+        between vectors from `x` and `y`.
+
+    Examples
+    --------
+    The cosine similarity between two equivalent vectors will be 1. The cosine
```

Comment:
Can you pls render the docs and check if it is happening properly.

Reply:
May I know how to render the doc?

Reply:
https://github.com/deepchem/deepchem/tree/master/docs

You can follow instructions from here.